DbCache_Plugin.php

```diff
@@ -160,7 +160,7 @@ public function on_comment_change( $comment_id ) {
 	}
 
 	/**
-	 * Comment status action
+	 * Comment status action triggered by approve/spam/trash comment quick-links under items on the Comments admin page.
```

Comment:
Is this the only time this method is triggered?

Or is it triggered any time comment status is changed?

Reply:
It's trigger anytime the status is changed via the quick-links under the items listed on the Comments admin page

Reply:
Correct me if I'm wrong, but I believe your description is inaccurate.

wp_set_comment_status is called more often than just when a quick-link changes the comment:
https://github.com/WordPress/WordPress/search?q=wp_set_comment_status

Reply:
You are not incorrect. The hook is called more often and by varying actions. I've updated the comment block and included a WP documentation link for the wp_set_comment_status hook